src/libkrun/build.rs

```diff
@@ -5,5 +5,10 @@ fn main() {
         std::env::var("CARGO_PKG_VERSION_MAJOR").unwrap()
     );
     #[cfg(target_os = "macos")]
+    println!(
+        "cargo:rustc-cdylib-link-arg=-Wl,-install_name,libkrun.{}.dylib,-compatibility_version,1.0.0,-current_version,{}.0.0",
+        std::env::var("CARGO_PKG_VERSION_MAJOR").unwrap(), std::env::var("CARGO_PKG_VERSION_MINOR").unwrap()
```

Comment:
Reading `ld`'s manual, I'd say `compatibility_version` should be `MAJOR.0.0`, and `current_version` `MAJOR.MINOR.RELEASE`.